tests/test_parse_mimetype_db.py

```diff
@@ -0,0 +1,64 @@
+import io
+import json
+import pytest
+
+from freezeyt.util import parse_mimetype_db
+
+TESTCASES = {
+    "catch_jpeg": (
+        {
+            "application/3gpdash-qoe-report+xml": {
+                "source": "iana",
+                "charset": "UTF-8",
+                "compressible": True
+            },
+            "image/jpeg": {
+                "source": "iana",
+                "compressible": False,
+                "extensions": ["jpeg","jpg","jpe"]
+            }
+        },
+        {"image/jpeg": ["jpeg","jpg","jpe"]}
+    ),
+    "no_catch": (
+        {
+            "audio/melp600": {"source": "iana"},
+            "audio/mhas": {"source": "iana"}
+        },
+        {}
+    ),
+    "many_catch": (
+        {
+            "audio/wav": {
+                "compressible": False,
+                "extensions": ["wav"]
+            },
+            "audio/wave": {
+                "compressible": False,
+                "extensions": ["wav"]
+            },
+            "audio/webm": {
+                "source": "apache",
+                "compressible": False,
+                "extensions": ["weba"]
+            },
+        },
+        {
+            "audio/wav": ["wav"],
+            "audio/wave": ["wav"],
+            "audio/webm": ["weba"]
```

Comment:
Original mime_db for gh pages contains, separately same extensions for different mime-types